beam/README.md

```diff
@@ -123,9 +128,9 @@ METADATA+=",flink-start-yarn-session=true"
 METADATA+=",flink-snapshot-url=${FLINK_SNAPSHOT}"
 
 gcloud dataproc clusters create "${CLUSTER_NAME}" \
-  --initialization-actions="${INIT_ACTIONS}" \
-  --image-version="1.2" \
-  --metadata="${METADATA}"
+    --initialization-actions "${INIT_ACTIONS}" \
+    --image-version "1.2" \
```

Comment:
Why 1.2?

Reply:
I don't think that we got Beam to work on newer Dataproc images.